### ISSUE TITLE
Update not working right

### ISSUE THREAD

**Issue by rbarbier1967** [AzuraCast/AzuraCast#5925]
### Installation Method

Docker Installation

### AzuraCast Release Channel

Rolling Release Channel

### What happened?

When I go to update Azuracast it goes through the motions, though never see it pull the update and then when it supposedly recreates the azuracast container it dont. I have to use the install method to return back to running status. 

Also been having issues with Directory submissions to Icecast and shout cast, it sends https://shokanradio.com:6000/radio instead of the webproxy url and the url it sends does not connect, gets the error 

This site can’t be reachedThe webpage at https://shokanradio.com:6000/radio might be temporarily down or it may have moved permanently to a new web address.
ERR_UNSAFE_PORT

I removed certbot and any kind of https records outside of azuracast, even when back to the originator of the dns and dropped cloudlare to trry and fix the problems. 

Right now the upgrade issue is my main concern, but would love to fix the connection issues / dir listing issues eventually

**Comment by rbarbier1967** [comment]
Fixed The Directory issues, seems port 6000 though unused was not working. lol well still facing the update issue though. Let me know if you need any of the log data and which.

**Comment by rbarbier1967** [comment]
Okay, I tried something that may had fixed the update issue, going to close this ticket after this but will say what I did so if someone else runs across this problem. I basically swapped from rolling release to stable release and back . though during the updat under stable release i had some errors, but once i returned to rolling release they didnt show. Seems to have the update issue fixed for now, will reopen a ticket if i have any further issues. Though may open a ticket for the shoutcast directory issue.
